### PR TITLE
Fixes block name test failures

### PR DESCRIPTION
**Pull request by sydneyli**
The interaction of #66 and #67 led to some test failures. Maybe we can set up nightly runs for CI in the future?

### CODE REVIEW

**Comment by etingof** [comment]
> Maybe we can set up nightly runs for CI in the future?

I just added an unconditional [daily build](https://travis-ci.org/etingof/apacheconfig) to Travis CI.

File: tests/unit/test_lexer.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ def testEmptyElementTagsDisabled(self):
 """
         ApacheConfigLexer = make_lexer(**options)
         tokens = ApacheConfigLexer().tokenize(text)
-        self.assertEqual(tokens, ['block/', '\n', 'block /', '\n', 'block hello/', '\n'])
+        self.assertEqual(tokens, [('block/',), '\n', ('block', ' ', '/'), '\n', ('block', ' ', 'hello/'), '\n'])
```
Comment (etingof) [inline tests/unit/test_lexer.py]: Interesting - how a 109-char long line passed our flake8 job...?

Comment (etingof) [reply tests/unit/test_lexer.py]: Lets merge these two PRs right away to make it quick and easy!

If you'll have a chance to get the CI failing on PEP8 issues - that would be great!

**Comment by sydneyli** [comment]
> Interesting - how a 109-char long line passed our flake8 job...?
Hmm, I think `flake8` is running on the source repo but not the `tests` repo.

**Comment by etingof** [comment]
Ah, right!

I can probably make it running on tests tomorrow. That may cause massive collisions with the upcoming PRs if you have some. Do you? ;-)

**Comment by sydneyli** [comment]
I do have the large interface PR incoming, but luckily it doesn't really touch the existing tests-- just adds more.

**Comment by etingof** [comment]
Alright then!

**Pull request merged**